misc(init-generator) cli improvements & refactor (#817)
webpack/webpack-cli: packages/webpack-scaffold/README.md

### Input

Param: `name<String>, message<String>, [default<String>]`

Creates an Input from Inquirer

```js
const Input = require("@webpack-cli/webpack-scaffold").Input;

Input('entry', 'what is your entry point?', 'src/index')
```

### InputValidate

Param: `name<String>, message<String>, [validate<Function>, default<String>]`

Creates an Input from Inquirer

```js
const InputValidate = require('@webpack-cli/webpack-scaffold').InputValidate;

const validation = (value) => {
    if(value.length > 4) {
        return true;
    } else {

>         return 'Wow, that was short!'

Reviewer: Give the user actions to improve explicitly instead

	if (value.length > 4) {
		return true;
	} else {

> 		return "Your answer must be longer than 4, try again";

Reviewer 2: ```suggestion
		return "Your answer must be longer than 4 characters, try again";
```

Author: Thanks! :)

> 		return "The entry point must be longer than 4, try again";

Reviewer: ```suggestion
		return "Your answer must be longer than 4 characters, try again";
```
